builders_guide.md

```diff
@@ -0,0 +1,551 @@
+2. Make a GitHub [account](https://github.com/).
+3. Clone the repo and install submodules and dependencies: 
+    1. If you’re here to experiment with a local game and learn about the codebase, follow the [Quickstart for running a local game](README.md#quickstart-for-running-a-local-game) instructions.
+    2. If you’re here to deploy a community round to production, follow the [If you plan to make changes to darkforest-local](README.md#if-you-plan-to-make-changes-to-darkforest-local) instructions.
+    3. Don’t run `yarn start` just yet - we will practice testing the smart contracts before running a local game in Quest 1.
+
+# Quest 1: Local Game
+
+- We are going to learn how run a local game of Dark Forest with no changes to the current configuration.
+
+## 1.0 Run Hardhat tests
+
+- Read a quick [overview](dapp.md#what-is-hardhat) of the Hardhat deployment environment.
+
+- This is not strictly necessary, but a good sanity check to make sure that the current smart contracts are passing the existing tests:  
+	`yarn workspace eth test`
+	
+- You should see output like this:
+    
+    ![Hardhat Test](img/test.png)
+    
+    ![Hardhat Pass](img/pass.png)
+    
+- If everything looks good, the Solidity backend is ready to go.
+
+
+## 1.1 Run a local game
+
+- This is another sanity check to make sure the client and game have no obvious bugs.
+- Run `yarn start`
+    - You should have already run `yarn` in Step 0.3. Run `yarn` if you didn't already.
+- In the game, test out the play. If you have any problems, check out the [Troubleshooting](#troubleshooting) section. Find a home planet, make a few moves, reveal a planet, etc. If everything looks good, you can be reasonably confident that everything will work in production.
+- To stop the local Hardhat network and kill the client, type `CTRL C` (^C) in your terminal.
+
+# Quest 2: Local game with custom configuration
+
+- For this quest, you will configure the universe to be a race to the center with one objective: *Be the first person to capture the Level 9 Planet at coordinate (0,0).*
+
+## 2.0 Understand `darkforest.toml`
+
+- Now we are going to make some changes to the game via the settings file. This file is called `darkforest.toml` and lives in the [eth](https://github.com/darkforest-eth/eth/) repository.
+- The `darkforest.toml` is useful because it allows you to configure a large number game parameters without writing a single line of code. Understanding what each parameter means is also a great way to get acquainted with the codebase.
+- Read the explanation of each setting in this [gist](https://gist.github.com/cha0sg0d/2d4d9b95a5f90c1b0e285b601b850827).
+  - *Note: Parameters that affect shrinking are not in the base codebase, so can't be used for this tutorial*.
+
+## 2.1 Configure `darkforest.toml`
+
+- Duplicate the existing the `darkforest.toml` file.
+- Rename it `darkforest.local.toml`
```

Comment:
```suggestion
- Rename it `darkforest.localhost.toml`
```
It uses the full network name, which I believe is always `localhost` in the hardhat node scenario.